3_cv_basics/4_morphology.ipynb

```diff
@@ -1,222 +1,237 @@
 {
```

Comment:
<p>Instead of listing down the figure notations, add them as a image subtitle to their respective images</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Pixels_Seminar/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>